src/app/shared/doc-viewer/doc-viewer.ts

```diff
@@ -19,16 +19,16 @@ import {Subscription} from 'rxjs';
 import {take} from 'rxjs/operators';
 import {ExampleViewer} from '../example-viewer/example-viewer';
 import {HeaderLink} from './header-link';
+import {DomPortalOutlet} from '@angular/cdk/portal/dom-portal-outlet';
```

Comment:
As mentioned on Slack. This import seems the culprit. Any reason why you use a deep import?

Reply:
That's what WebStorm defaulted to when auto importing 😞